Common Data Binding: Add the Update Data article (#260)
DevExpress/devextreme-documentation: concepts/70 Data Binding/03 Update Data/05 DevExtreme DataSource/02 Data Shaping.md

> Data shaping is implemented by the [DataSource](/api-reference/30%20Data%20Layer/DataSource '/Documentation/ApiReference/Data_Layer/DataSource/') component and its methods. To call them, get a **DataSource** instance from your widget using the [getDataSource()](/api-reference/10%20UI%20Widgets/DataHelperMixin/3%20Methods/getDataSource().md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#getDataSource') method. Alternatively, you can use a standalone instance saved in a constant/component property when you created the **DataSource**.

Reviewer: ```suggestion
The [DataSource](/api-reference/30%20Data%20Layer/DataSource '/Documentation/ApiReference/Data_Layer/DataSource/') component and its methods implement data shaping. To call these components, use the [getDataSource()](/api-reference/10%20UI%20Widgets/DataHelperMixin/3%20Methods/getDataSource().md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#getDataSource') method to get a **DataSource** instance from your widget . You can also use a standalone instance saved in a constant/component property when you created the **DataSource**.
```

> In the following code, we get a **DataSource** instance using both approaches and call one of the data shaping methods&mdash;[filter(filterExpr)](/api-reference/30%20Data%20Layer/DataSource/3%20Methods/filter(filterExpr).md '/Documentation/ApiReference/Data_Layer/DataSource/Methods/#filterfilterExpr'). Such methods only set up data shaping settings. To apply them, we call the [load()](/api-reference/30%20Data%20Layer/DataSource/3%20Methods/load().md '/Documentation/ApiReference/Data_Layer/DataSource/Methods/#load') method.

Reviewer 2: >Such methods only set up data shaping settings.

Shouldn't it be 'set data shaping settings'?

Reviewer 3: I would suggest adding a code that processes the result

  dataSource.load().then((items)=>{
        // process data here
  })

Author: I'm not sure. Let's leave it to editors.

> In the following code, we get a **DataSource** instance using both approaches and call one of the data shaping methods&mdash;[filter(filterExpr)](/api-reference/30%20Data%20Layer/DataSource/3%20Methods/filter(filterExpr).md '/Documentation/ApiReference/Data_Layer/DataSource/Methods/#filterfilterExpr'). Such methods only set up data shaping settings. To apply them, we call the [load()](/api-reference/30%20Data%20Layer/DataSource/3%20Methods/load().md '/Documentation/ApiReference/Data_Layer/DataSource/Methods/#load') method.

Reviewer: ```suggestion
In the following code, we use both approaches to get a **DataSource** instance and call one of the data shaping methods&mdash;[filter(filterExpr)](/api-reference/30%20Data%20Layer/DataSource/3%20Methods/filter(filterExpr).md '/Documentation/ApiReference/Data_Layer/DataSource/Methods/#filterfilterExpr'). These methods only specify data shaping settings. To apply them, we call the [load()](/api-reference/30%20Data%20Layer/DataSource/3%20Methods/load().md '/Documentation/ApiReference/Data_Layer/DataSource/Methods/#load') method.
```

Reviewer: "To apply them" What does 'them' refer to here?
